core/modules/system/system.admin.inc

```diff
@@ -696,14 +699,28 @@ function system_sort_themes($a, $b) {
  * Build a table row for the system modules page.
  */
 function _system_modules_build_row($info, $extra) {
+  // Prepare the tag list by ensuring all tags are lower case.
+  $tags = array();
+  if (isset($info['tags'])) {
+    foreach ($info['tags'] as $tag) {
+      $tags[] = strtolower($tag);
+    }
+  }
+  if (isset($info['package'])) {
+    $tags[] = strtolower($info['package']);
```

Comment:
I don't see any reason to cast these to lower case. If we want to keep this, `backdrop_strtolower()` should be used. But IMO, we should just remove the conversion entirely.

Reply:
But we should run everything to `check_plain()` to ensure no encoding problems. e.g. a tag might contain an ampersand or quote character.

Reply:
Fixed and fixed.